datasets/gov/vrk/rinkeju_demografiniai_duomenys.csv

```diff
@@ -0,0 +1,31 @@
+,datasets/gov/vrk/rinkeju_demografiniai_duomenys,,,,,,,,,,,,Rinkėjų demografiniai duomenys,"Vyriausiosios rinkimų komisijos Rinkėjų sąrašų tvarkymo posistemė kaupia rinkėjų balsavimo žymėjimus. Duomenys kaupiami nuo 2011-12-29 ir yra atnaujinami. Duomenų rinkinys sudarytas iš 4 atskirų lentelių apie rinkėjus, rinkėjų žymėjimus, rinkimų apygardas ir vykdomus rinkimus. Lentelėse yra įtraukti išorinių raktų stulpeliai suteikti galimybę apjungti jose esančius duomenis. Turint klausimų apie duomenų rinkinį, prašome kreiptis į Valstybės duomenų valdysenos atvėrimo komandą elektroniniu paštu atverimas@stat.gov.lt."
+,,,,,,prefix,spinta,,,,,https://github.com/atviriduomenys/spinta/issues/,,
+,,,,,,,manifest,,,,,https://github.com/atviriduomenys/manifest/issues/,,
+,,,,,,,,,,,,,,
+,,,,Rinkejas,,,rinkejo_asm_k,,,4,,,Rinkėjai,"Balso teisę turinčių rinkėjų lentelė. Vienas įrašas atitinka vieną rinkėją. Lentelė sudaryta iš rinkėjo pseudonimizuoto asmens kodo, lyties ir gimimo metų. ~12% rinkėjų gimimo metai yra užtriukšminti siekiant užtikrinti konfidencialumą."
+,,,,,rinkejo_asm_k,string,,,,4,open,,Rinkėjo pseudonimizuotas asmens kodas,
+,,,,,rinkejo_lytis,string,,,,4,open,,Rinkėjo lytis,
+,,,,,,enum,,,"""V""",,,,Vyras,
+,,,,,,,,,"""M""",,,,Moteris,
+,,,,,,,,,,,,,,
+,,,,,rinkejo_gim_m,date,Y,,,4,open,,Rinkėjo gimimo metai,
+,,,,,,,,,,,,,,
+,,,,Zymejimas,,,vda_prime_key,,,4,,,Rinkėjų žymėjimai,"Rinkėjų balsų žymėjimų lentelė. Vienas įrašas atitinka vieno rinkėjo konkrečiais rinkimais (ne)balsavimo žymėjimą. Lentelė sudaryta iš pseudonimizuoto rinkėjo asmens kodo, vykdomų rinkimų turo ir apygardos, kurioje nubalsuota, identifikatorių.  "
+,,,,,vda_prime_key,string,,,,4,open,,Unikalus identifikatorius,
+,,,,,rinkejo_asm_k,string,,,,4,open,,Rinkėjo pseudonimizuotas asmens kodas,
+,,,,,rinkimu_turo_id,string,,,,4,open,,Vykdomų rinkimų turo identifikatorius,
+,,,,,faktines_apygardos_id,string,,,,4,open,,"Apygardos, kurioje nubalsuota, identifikatorius",
+,,,,,,,,,,,,,,
+,,,,Apygarda,,,apygardos_id,,,4,,,Rinkimų apygardos,Rinkimų apygardų lentelė. Vienas įrašas atitinka vieną apygardą tam tikrų rinkimų metu. Lentelė sudaryta iš apygardos identifikatoriaus ir pavadinimo.
+,,,,,apygardos_id,string,,,,4,open,,Apygardos identifikatorius,
+,,,,,apygardos_pav,string,,,,4,open,,Apygardos pavadinimas,
+,,,,,,,,,,,,,,
+,,,,RinkimuTuras,,,vda_prime_key,,,4,,,Rinkimų turai,"Vykdomų rinkimų lentelė. Vienas įrašas atitinka vykdomų rinkimų ir jų turų kombinaciją. Lentelė sudaryta iš vykdomų rinkimų identifikatoriaus ir pavadinimo, vykdomų rinkimų turo identifikatoriaus, numerio, dienos ir, ar turas yra einamasis."
+,,,,,vda_prime_key,string,,,,4,open,,Unikalus identifikatorius,
+,,,,,rinkimu_id,string,,,,4,open,,Vykdomų rinkimų identifikatorius,
+,,,,,rinkimu_pav,string,,,,4,open,,Vykdomų rinkimų pavadinimas,
+,,,,,rinkimu_turo_id,string,,,,4,open,,Vykdomų rinkimų turo identifikatorius,
+,,,,,rinkimu_turo_nr,integer,,,,4,open,,Vykdomų rinkimų turo numeris,
```

Comment:
Ar tikrai tinkamas tipas "integer" rinkimų turo numeriui? Pagal vadovą, tai tada turėtų būti matavimo vienetas 4 brandos lygiui. Gal tiktų ir string?

Reply:
`string` netiktų, nes, kadangi tai numeris, VDV IS naudojame `long` tipą. Variantas galėtų būt žymėti `U` vienetą.